containers/Dockerfile.EESSI-client-pilot-centos7

```diff
@@ -1,5 +1,5 @@
 ARG cvmfsversion=2.8.1
-ARG fuseoverlayfsversion=0.4.1
+ARG fuseoverlayfsversion=0.3
```

Comment:
@bedroge Should we add a comment here to clarify why we're sticking to an older one?
A reference to https://github.com/containers/fuse-overlayfs/issues/232?